file-upload/backend/src/main/scala/io/udash/demos/files/jetty/DemoFileUploadServlet.scala

```diff
@@ -0,0 +1,27 @@
+package io.udash.demos.files.jetty
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import java.io.{File, InputStream}
+import java.nio.file.Files
+import java.util.UUID
+
+import io.udash.demos.files.UploadedFile
+import io.udash.demos.files.rpc.ClientRPC
+import io.udash.demos.files.services.FilesStorage
+import io.udash.rpc._
+
+class DemoFileUploadServlet(uploadDir: String) extends FileUploadServlet(Set("file", "files")) {
+  new File(uploadDir).mkdir()
+
+  override protected def handleFile(name: String, content: InputStream): Unit = {
+    val targetName: String = s"${UUID.randomUUID()}_$name"
```

Comment:
Could this `name` be literally anything?
I think that a file system could impose some limitations on file name, and it could be a good idea to pass it through an escaper.